src/bundle/Resources/public/scss/_side-panel.scss

```
@@ -0,0 +1,47 @@
+.ibexa-side-panel {
+    background-color: $ibexa-color-white;
+    padding: calculateRem(8px) 0;
+    width: calculateRem(516px);
+    height: calc(100vh - #{calculateRem(73px)});
```

I think we do not need `#{}` here and below
```suggestion
    height: calc(100vh - calculateRem(73px));
```